### ISSUE TITLE
有关模型inference的问题，Question about inferencing the model.

### ISSUE THREAD

**Issue by shihanmax** [hanjiale/HCRP#2]
作者您好！ 感谢您出色的实验，这里有一个有关模型推理的问题：

您的模型实现中，模型推理时需要传入`rel_text`，即关系类型（及其描述），fewRel的测试集中，support set并没有给出每个support instance的关系类型，想请问一下，您在测试集上做推理时，是如何处理support set对应的关系类型的呢？

```Python
# in model.py

def forward(self, support, query, rel_text, N, K, total_Q, is_eval=False):
        """
        :param support: Inputs of the support set. (B*N*K)
        :param query: Inputs of the query set. (B*total_Q)
        :param rel_text: Inputs of the relation description.  (B*N)
        :param N: Num of classes
        :param K: Num of instances for each class in the support set
        :param total_Q: Num of instances in the query set
        :param is_eval:
        :return: logits, pred, logits_proto, labels_proto, sim_scalar
        """

```

**Comment by hanjiale** [comment]
Thank you for your attention to our work! Actually, the label information is given in the FewRel test set. You can check here for details https://worksheets.codalab.org/worksheets/0x224557d3a319469c82b0eb2550a2219e (with relation id). The given relation list corresponds to the support set in order.
